packages/core/src/awsService/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -919,6 +943,11 @@ export function isJsonPolicyLanguage(document: string) {
     return policyLanguageFileTypes.some((t) => document.endsWith(t))
 }
 
+export function getProfileName(): string | undefined {
+    // We neeed to split the name on 'profile:' to extract the correct profile name
+    return globals.awsContext.getCredentialProfileName()?.split('profile:')[1]
```

Comment:
Is this a constant format? I am trying to understand who enforces this format?

Reply:
Also, can we add an error message on the backend to determine any incorrect format here?

Reply:
Toolkits uses the format in displaying the profile name to users after authenticating through the extension. It's stored with `profile:` prepended in the Credentials object.